examples/Client/DistributedLockApi/UnLockApplication/Program.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using System.Threading;
+using Dapr.Client;
+
+namespace DistributedLockApi
+{
+    public class Program2
+    {
+        [Obsolete]
+        public static async Task Main(string[] args)
```

Comment:
Nit: your file path no longer matches the name you give this in the run command.